### PR TITLE
151 login test

### PR DESCRIPTION
**Pull request by IuryNogueira**

### CODE REVIEW

File: frontend/cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (ValmirJuniorr) [inline frontend/cypress/fixtures/example.json:5]:
```suggestion
}

```

**Review by ValmirJuniorr — APPROVED** [review]
LGTM.

File: frontend/cypress/integration/login/login.spec.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  });
+
+  it('should show register button', () => {
+    cy.get('#btn-register');
+  });
+
+  it('should show and type email', () => {
+    cy.get('#email')
+      .type('admin@gmail.com');
+  });
```
Comment (andersonmalheiro) [inline frontend/cypress/integration/login/login.spec.js:15]: I think you can add an statement checking if the input has the value typed, like this:

```suggestion
  it('should show and type email', () => {
    cy.get('#email')
      .type('admin@gmail.com')
      .should('have.value', 'admin@gmail.com');
  });
```

Same thing for the other cases below

File: frontend/cypress/integration/login/login.spec.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      .type('invaliduser@gmail.com');
+
+    cy.get('#password')
+      .type('maybeiscorrectpassword');
+
+    cy.get('#btn-login')
+      .click();
+
+    cy.url().should('not.be.include', '/main');
+  });
```
Comment (andersonmalheiro) [inline frontend/cypress/integration/login/login.spec.js:33]: You could also intercept the request to check if it returns an error
https://docs.cypress.io/api/commands/intercept#Intercepting-a-response

**Review by andersonmalheiro — CHANGES_REQUESTED** [review]

**Review by edimossilva — APPROVED** [review]
LGTM!!! 

Did you create an issue to add it to CI?

**Comment by IuryNogueira** [comment]
> LGTM!!!
> 
> Did you create an issue to add it to CI?
> 
> your commits looks strange =z

this commit? https://github.com/espoo-dev/espoo-dev/pull/182/commits/13009b5c1f792fb5391590e9f927eea059c9ca2b

**Comment by IuryNogueira** [comment]
this pull request too fix #209

**Pull request merged**